main/main.cpp

```diff
@@ -2658,7 +2672,7 @@ bool Main::start() {
 				parsed_pair = false;
 			}
 			if (parsed_pair) {
-				i++;
+				++i;
 			}
```

Comment:
As the return value isn't used this doesn't really optimise anything, anything but an incredibly primitive compiler will resolve this

Reply:
alright i changed that back to normal